src/commands/economy/GiveCommand.js

```diff
@@ -10,175 +43,118 @@ module.exports = class GiveCommand extends Command {
     });
   }
 
-  async giveStar(user, user2, valor, ctx, mencionado) {
-    if (valor > user.estrelinhas)
-      return ctx.reply(
-        'error',
-        `${ctx.locale('commands:give.poor')} ${ctx.locale('commands:give.stars')}`,
-      );
+  async run(ctx) {
+    const selectedOption =
+      ctx.args[0] &&
+      validArgs.find((option) => option.arguments.includes(ctx.args[0].toLowerCase()));
+    if (!selectedOption) return this.replyInvalidArgsError(ctx);
 
-    await this.client.database.Users.updateOne({ id: user.id }, { $inc: { estrelinhas: -valor } });
-    await this.client.database.Users.updateOne({ id: user2.id }, { $inc: { estrelinhas: valor } });
+    const to = ctx.message.mentions.users.first();
+    if (!to) return this.replyBadUsageError(ctx);
+    if (to.id === ctx.message.author.id) return this.replyForYourselfError(ctx);
 
-    ctx.reply(
-      'success',
-      `${ctx.locale('commands:give.transfered', { value: valor, emoji: '⭐' })} ${mencionado}`,
-    );
+    const input = ctx.args[2];
+    if (!input) return this.replyBadUsageError(ctx);
+
+    const value = parseInt(input.replace(/\D+/g, ''));
+    if (!value || value < 1) return this.replyInvalidValueError(ctx);
+
+    const toData = await this.client.database.repositories.userRepository.findOrCreate(to.id);
```

Comment:
Redundant: `this.client.repositories` is a get to `this.client.database.repositories`

You were who created `get` and already forgot... you need to use Menhera's code more

Reply:
nossa

Reply:
LUX SQL LOVER IS REAL — Today at 2:39 AM
NAO ERA PRA SER GROSSA
DESUCLPA
era pela piada
:MenheraSad:

Reply:
I ONLY CHANGED THE VARIABLE NAME THIS LINE

Reply:
NOSSA IS NOT AN ENGLISH WORD I AM FICANDO CRAZY WITH NOSSAS CONVERSAR AQUI IN THE GITHUB

Reply:
> LUX SQL LOVER IS REAL — Today at 2:39 AM
> NAO ERA PRA SER GROSSA
> DESUCLPA
> era pela piada
> :MenheraSad:

calma, eu to brincando

Reply:
![image](https://user-images.githubusercontent.com/59155752/125397204-fffe2a00-e383-11eb-949b-03052d64df23.png)

Acho lindo isso

Reply:
> NOSSA IS NOT AN ENGLISH WORD I AM FICANDO CRAZY WITH NOSSAS CONVERSAR AQUI IN THE GITHUB

IWADHAUFHAWUFHADWHIWH